### ISSUE TITLE
Numerical Instability for Quantile Loss ?

### ISSUE THREAD

**Issue by akrishnamo** [catboost/catboost#1137]
Problem: Slight variations in Quantile Loss percentage creating huge swings in predictions

catboost version: 0.20.2
Operating System, CPU: Google Collab
GPU: None

I am trying to do a quantile forecast using CatBoost, I have three models for 10%, 50%, 90%

modelxyQ10=CatBoostRegressor(iterations=20,learning_rate=1, depth=3, cat_features=[0,1], loss_function='Quantile:alpha=0.10')
modelxyQ50=CatBoostRegressor(iterations=20,learning_rate=1, depth=3, cat_features=[0,1], loss_function='Quantile:alpha=0.50')
modelxyQ90=CatBoostRegressor(iterations=20,learning_rate=1, depth=3, cat_features=[0,1], loss_function='Quantile:alpha=0.90')

![image](https://user-images.githubusercontent.com/4354395/71956819-20d74380-3212-11ea-8ea7-ecd64b60cc48.png)

Notice that the 10Percentile is above the 90th

Now, by simply changing the first loss function to quantile of 11%, note the change (pink line)
modelxyQ10=CatBoostRegressor(iterations=20,learning_rate=1, depth=3, cat_features=[0,1], loss_function='Quantile:alpha=0.11')

![image](https://user-images.githubusercontent.com/4354395/71956987-904d3300-3212-11ea-833d-800124d6d82d.png)

Is this a numerical stability issue, or something else ?

**Comment by LyzhinIvan** [comment]
Hello, @akrishnamo !
Could you provide you dataset? What shape of your data? 20 iterations can be too small to fit the learn points.

**Comment by andrey-khropov** [comment]
As additional information to reproduce the issue has not been provided for a long time I am closing this issue.
